.github/workflows/PerfTest.yml

```diff
@@ -1,8 +1,10 @@
 name: Kafka Connector Performance Test
 
 on:
-  schedule:
-    - cron:  '0 6 * * *'
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: '**'
```

Comment:
How long does the test take? I am wondering if we can keep this for good if it takes < 30 minutes

Reply:
(Honestly, I am running this for first time.)
Creating PRs is my goto way to run GH Actions and figure out if code is working or not. (Honestly, that is the fastest way right now to test anything)

Reply:
But, I will keep this temporarily and once they start running, I will revert this back to cron expression

Reply:
That is sad :(. Maybe we can add a job for this in Jenkins so that we can schedule this adhoc? Can you file a JIRA, I might have bandwidth in the next couple of weeks to pick it up

Reply:
I did dig a bit this morning, but need to spend more time. There seem to exist Tests repo already which has this perf_test code. https://github.com/snowflakedb/Tests
And there is a Jenkins associated but it seems it is dormant. 
https://jenkins.int.snowflakecomputing.com/job/CLIENTS_PERF/

Also, I think you can run this add hoc from github action.

Reply:
Example: https://github.com/snowflakedb/snowflake-kafka-connector/actions/runs/843190933
Just click re-run.

Reply:
The test have passed after some modifications and play around. I will push this and send another PR to revert to cron job. (It takes around 2 hours to complete it)

Reply:
Yes please change it to a scheduled daily run before checking in. Blocking PRs for 2 hours is not good.

Reply:
Changed